tools/goseq_1_22_0/test-data/dge_list.tab

```diff
@@ -0,0 +1,1000 @@
+ENSG00000140459	False
+ENSG00000236824	False
+ENSG00000162526	False
```

Comment:
I would make the file small and use `contains`. I suspect this is changing from version to version and `contains` may also be more stable.